SonarQube.TeamBuild.PreProcessor/Interfaces/IPropertiesFetcher.cs

```diff
@@ -17,6 +17,6 @@ public interface IPropertiesFetcher
         /// </summary>
         /// <param name="ws">SonarQube Web Service instance</param>
         /// <param name="sonarProjectKey">The key of the SonarQube project for which the properties should be fetched</param>
-        IDictionary<string, string> FetchProperties(SonarWebService ws, string sonarProjectKey);
+        IDictionary<string, string> FetchProperties(SonarWebService ws, string sonarProjectKey, ILogger logger);
```

Comment:
Updated the interface to add basic logging around fetching properties from the SonarQube server.